src/widgets/swap-widget/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {useCallback} from 'react';
+
+import {observer} from 'mobx-react';
+import {Image, View} from 'react-native';
+
+import {Color} from '@app/colors';
+import {Spacer, Text, TextVariant} from '@app/components/ui';
+import {ShadowCard} from '@app/components/ui/shadow-card';
+import {WidgetHeader} from '@app/components/ui/widget-header';
+import {awaitForWallet, createTheme} from '@app/helpers';
+import {useTypedNavigation} from '@app/hooks';
+import {I18N, getText} from '@app/i18n';
+import {Wallet} from '@app/models/wallet';
+import {HomeStackRoutes} from '@app/route-types';
+
+export const SwapWidget = observer(() => {
+  const navigation = useTypedNavigation();
+
+  const onPress = useCallback(async () => {
+    const address = await awaitForWallet({
+      wallets: Wallet.getAll(),
+      title: I18N.selectAccount,
+    });
+    navigation.navigate(HomeStackRoutes.Swap, {address});
+  }, [navigation]);
+
+  return (
+    <ShadowCard style={styles.wrapper} onPress={onPress}>
+      <View style={styles.image}>
```

Comment:
Height and width should be applied to image iteself -- there is no need to have extra wrapper